migrations/20240520201323_add_google_account_id.up.sql

```diff
@@ -0,0 +1,7 @@
+-- Add up migration script here
+
+ALTER TABLE members
+    ADD COLUMN google_account_id VARCHAR(255) NULL;
```

Comment:
As per discussion on discord: Let's put a `UNIQUE` constraint here as a failsafe for raceconditions that allow adding the same google account to multiple pointercrate accounts